bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -22,10 +22,11 @@
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 
-	<channel-type id="light_level" advanced="true">
+	<channel-type id="light_level">
```

Comment:
Why this change?

Reply:
Because this is the PRIMARY channel for a light level sensor, and should not be hidden away.